### DISCUSSION TITLE
Per camera quality setting?

### DISCUSSION THREAD

**Discussion by backslashV** [mrlt8/docker-wyze-bridge#199] [Ideas]
Is it possible to have per camera quality setting?

**Comment by mrlt8** [comment]
We're already doing this for the doorbell, so it should be possible.

The only downside is that it probably wouldn't work in home assistant mode because of the schema checks.

**Comment by mrlt8** [comment]
Added in v1.0.2.

Can set quality for a specific camera with `QUALITY_CAM_NAME=HD200`

**Reply by backslashV** [reply]
Thanks a lot. Will give it a try. Does this value override what's set in the app?

**Reply by mrlt8** [reply]
Yes, it uses the same stream as the app, so any changes here should be reflected in the app and vice versa.

**Comment by Ellerbestyle** [comment]
I am confused as to where I put this? If I am using this as an add-on for HA, what file do I add this to?

**Reply by backslashV** [reply]
In the add-on's configuration tab. Click on _edit in yaml_ and add it there.
